### DISCUSSION TITLE
Elixir Phoenix Liveview

### DISCUSSION THREAD

**Discussion by pulley871** [shadcn-ui/ui#5009] [Q&A]
I love this package. Heavily use it at work.

Is there any plans into expanding into the  Phoenix Liveview framework. There is an extreme lack of good UI tools and I believe if this could get in there. It would be the most heavily used package in Liveview.

**Comment by SwannHERRERA** [comment]
I feel the same as you the ecosystem of the design tool is why I stay on react for the frontend of my project. I also understand that the project is based on react and svelte for the svelte version. The svelte version is not up-to-date with shadcn/ui. There is no Vue.js version. It will be very cool to have a component library like this on LiveView, but I think it's not the purpose of this project. If you have a way to use react inside of LiveView why not but I don't think it's compatible.
